### PR TITLE
dynamic-list-and-add-calendar

### PR DESCRIPTION
**Pull request by Manu7864z**
> created dynamic of activity 
> List is filtered by date 
> added React-calendar Package 
> choosing date in Calendar will update the list

> problem at the moment:  if you click the date e.g. 09.03.2023 you get 08.03.2023...

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,32 +1,61 @@
...
-  const options = {
+  /*  const options = {
     method: "GET",
     headers: {
-      "X-RapidAPI-Key": process.env.REACT_APP_API_KEY ,
+      "X-RapidAPI-Key": process.env.REACT_APP_API_KEY,
       "X-RapidAPI-Host": "weatherapi-com.p.rapidapi.com",
     },
-  };
+  }; */
```
Comment (stefankorder) [inline pages/_app.js:21]: Please delete this out commeted lines of code.

File: pages/_app.js [hunk 2]
```diff
@@ -1,32 +1,61 @@
...
 
-  const { data, error, isLoading } = useSWR(URL, fetcher, options);
+  const { data, error, isLoading } = useSWR(URL, fetcher);
   if (error) return <div>failed to load</div>;
   if (isLoading) return <div>loading...</div>;
   console.log(data);
```
Comment (stefankorder) [inline pages/_app.js:26]: Please delete this console log.

File: pages/index.js [hunk 3]
```diff
@@ -1,19 +1,92 @@
...
+  const [weekDay, setWeekDay] = useState(
+    new Date().toLocaleDateString("default", { year: "numeric" }) +
+      "-" +
+      new Date().toLocaleDateString("default", { month: "2-digit" }) +
+      "-" +
+      new Date().toLocaleDateString("default", { day: "2-digit" })
+  );
+
+  const [showCalendar, setShowCalendar] = useState(false);
+  console.log(weekDay);
```
Comment (stefankorder) [inline pages/index.js:18]: Please delete this console log.

**Review by stefankorder — CHANGES_REQUESTED** [review]
LGTM so far. But there are a few changes i suggest you.

**Review by landaluis — APPROVED** [review]
LGTM

**Pull request merged**